dbt_subprojects/tokens/models/prices/solana/prices_solana_tokens.sql

```diff
@@ -759,5 +759,7 @@ FROM
         ('dark-dark-eclipse', 'solana', 'DARK', '8BtoThi2ZoXnF7QQK1Wjmh2JuBw9FjVvhnGMVZ2vpump', 6),
         ('alch-alchemist-ai', 'solana', 'ALCH', 'HNg5PYJmtqcmzXrv6S9zP1CDKk5BgDuyFBxbvNApump', 6),
         ('zbcn-zebec-network', 'solana', 'ZBCN', 'ZBCNpuD7YMXzTHB2fhGkGi78MNsHGLRXUhRewNRm9RU', 6),
-        ('huma-finance', 'solana', 'HUMA', 'HUMA1821qVDKta3u2ovmfDQeW2fSQouSKE8fkF44wvGw', 6)
+        ('huma-finance', 'solana', 'HUMA', 'HUMA1821qVDKta3u2ovmfDQeW2fSQouSKE8fkF44wvGw', 6),
+        ('wen', 'solana', 'WEN', 'WENWENvqqNya429ubCdR81ZmD69brwQaaBYY6p3LCpk', 5),
+        ('zeus', 'solana', 'ZEUS', 'ZEUS1aR7aX8DFFJf5QjWj2ftDDdNTroMNGo8YoQm3Gq', 6)
```

Comment:
not valid coinpaprika API id values. plz find on coinpaprika, grab api ID provided there, and use in first column